builds/e2e/templates/longhaul-deploy-windows.yaml

```
@@ -15,6 +15,10 @@ parameters:
   longHaul.desiredModulesToRestartCSV: ''
   longHaul.restartIntervalInMins: ''
   analyzer.consumerGroupId: ''
+  analyzer.logAnalytic.enabled: ''
+  analyzer.logAnalytic.workspaceid: ''
```

fix casing for these parameter
```suggestion
  analyzer.logAnalytic.workspaceId: ''
```